charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -27,7 +27,7 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.password}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1265.service.core-compute-preview.internal/
```

Comment:
These values should be restored prior to merger.

Reply:
replaced comment with https://github.com/hmcts/ccd-data-store-api/pull/2064#pullrequestreview-1070880915